docs/release-notes/v13.1.0.md

```diff
@@ -0,0 +1,205 @@
+      },
+      "id": "my-route",
+      "labelIntl": {
+        "en": "My Extension",
+        "it": "La mia estensione"
+      },
+      "order": 200.0,
+      "locationId": "project",
+      "parentId": "my-menu-group"
+    },
+    {
+      "id": "my-menu-group",
+      "labelIntl": {
+        "en": "My Menu Group",
+        "it": "Il mio gruppo menu"
+      },
+      "locationId": "project",
+      "renderType": "category",
+      "order": 100.0,
+    },
+  ]
+}
+```
+
+it is possible to migrate to new format properties following the mapping explained by this table:
+
+| New schema properties | Legacy schema properties   |
+|-----------------------|----------------------------|
+| extensionId           | extensionId                |
+| name                  | name                       |
+| entry                 | entry                      |
+| type                  | extensionType              |
+| activationContexts    | contexts                   |
+| description           | description                |
+| permissions           | permission                 |
+| destination.id        | mainRoute.locationId       |
+| destination.path      | mainRoute.destinationPath  |
+| iconName              | mainRoute.icon.name        |
+| menu.id               | mainRoute.id               |
+| menu.labelIntl        | mainRoute.labelIntl        |
+| menu.order            | mainRoute.order            |
+| category.id           | mainRoute.parentId         |
+| category.labelIntl    | categoryRoute.labelIntl    |
+| category.order        | categoryRoute.order        |
+
+where:
+
+- **mainRoute** is the last route in the `routes` legacy properties that does not define renderType, so the route with `my-route` ID.
+- **categoryRoute** is the last route in the `routes` legacy properties that defines renderType as `category`. This is the custom category registered with the extension and it could be omitted in the request, in fact in this case the new `category` properties will have only the `id` defined taken by the `mainRoute.parentId`. In the example above there is a custom category and corresponding to the route with `my-menu-group` ID.
+
+After migration the new body format is the following:
+
+```json
+{
+  "extensionId": "my-extension-id",
+  "name": "My Extension",
+  "entry": "https://example.com/",
+  "type": "iframe",
+  "activationContexts": ["project"],
+  "description": "Example Extension",
+  "permissions": [],
+  "destination": {
+      "id": "project",
+      "path": "/",
+  },
+  "iconName": "PiProjectorScreenChartLight",
+  "menu": {
+    "id": "my-route",
+    "labelIntl": {
+      "en": "My Extension",
+      "it": "La mia estensione"
+    },
+    "order": 200.0,
+  },
+  "category": { 
+      "id": "my-menu-group",
+      "labelIntl": {
+        "en": "My Menu Group",
+        "it": "Il mio gruppo menu"
+      },
+      "order": 100.0,
+  }
+}
+```
+</details>
+
+### Bug Fix
+
+This version addressed the following bugs:

```

Comment:
```suggestion
This version addressed the following bugs:

* In Design Overview, when exporting Data Catalog records, documents that contained array values within `jsonSchemaType` and `ddlSchemaType` are now properly parsed and validated, so that they are returned in the resulting CSV file
```